README.md

```diff
@@ -34,6 +34,7 @@ Note: it might take 10-30 minutes depending on your machine and internet connect
 
 # List of created databases
 
+* Geode (port 10334)
```

Comment:
Can you change this to "Apache Geode".

Also please change "Cassandra" to "Apache Cassandra" (not sure when that mistake crept in!)

Reply:
Good call! I'm guessing the Cassandra one was me :(

Reply:
Sure i will. Actually i though about it but then deiced to follow the `Cassandra` example :)
I will fix both